File: Formacao_Python_Developer/Desafios_de_Codigo/02_Desafios_Intermediarios/03_Animal/codigo_resposta.py

a = input()
b = input()
c = input()

if a == 'vertebrado':
    if b == 'ave' and c == 'carnivoro':
        print('aguia')
    elif b == 'ave' and c == 'onivoro':
        print('pomba')
    elif b == 'mamifero' and c == 'onivoro':
        print('homem')
    elif b == 'mamifero' and c =='herbivoro':
        print('vaca')
elif a == 'invertebrado':

    if b == 'inseto' and c == 'hematofago':
        print('pulga')
    elif b == 'inseto' and c == 'herbivoro':
        print('lagarta')
    elif b == 'anelideo' and c == 'hematofago':
        print('sanguessuga')
    elif b == 'anelideo' and c =='onivoro':
        print('minhoca')

File: Bootcamp_Banco_PAN_Java_Developer/Desafios/01_Intermediarios/00_Animal/CodigoResposta.java
import java.io.IOException;
import java.util.Scanner;

public class CodigoResposta {

	public static void main(String[] args)  throws IOException {
		Scanner sc = new Scanner(System.in);
		
		String AN1,AN2,AN3;
		
		AN1 = sc.nextLine();
		AN2 = sc.nextLine();
		AN3 = sc.nextLine();

        if (AN1.equals("vertebrado")) {
            switch (AN2) {
                case "ave":
                    switch (AN3) {
                        case "carnivoro":
                            System.out.println("aguia");
                        break;
                        case "onivoro":
                            System.out.println("pomba");
                        break;
                    }
                break;
                case "mamifero":
                    switch (AN3) {
                        case "onivoro":
                            System.out.println("homem");
                        break;
                        case "herbivoro":
                            System.out.println("vaca");
                        break;
                    }
                break;

            }
        } else if (AN1.equals("invertebrado")) {
            switch (AN2) {
                case "inseto":
                    switch (AN3) {
                        case "hematofago":
                            System.out.println("pulga");
                        break;
                        case "herbivoro":
                            System.out.println("lagarta");
                        break;
                    }
                break;
                case "anelideo":
                    switch (AN3) {
                        case "hematofago":
                            System.out.println("sanguessuga");
                        break;
                        case "onivoro":
                            System.out.println("minhoca");
                        break;
                    }
                break;
            }
        }
    }
}
